app/src/main/res/values/strings.xml

```diff
@@ -34,6 +34,21 @@
     </string>
     <string name="will_rate">Sure, I\'ll help</string>
 
+    <string name="never_lost_your_data">Never Lose Your Data!</string>
```

Comment:
Instead of coupling the string name to the content of the string itself, it's better to describe how it's used IMHO, and you can see this in the other variable names (this is something we do at Instagram that I want to do more of). Here are the names we can use:

backup_and_restore_dialog_title
backup_and_restore_dialog_body
backup_and_restore_dialog_positive_button
backup_and_restore_dialog_negative_button